docs/faq/faq.md

```diff
@@ -40,3 +40,15 @@ There are some things you should consider before doing so:
   conflicts.
 
 TODO: Link to adoption use case once it gets on board.
+
+## Why Cluster API doesn't have the CRD YAML file?
+
+Kubernetes provides two methods to extend APIs: Custom Resource and Kubernetes API Aggregation Layer. For more detail, you can refer to [Extending the Kubernetes API](https://kubernetes.io/docs/concepts/extend-kubernetes/).
+
+Karmada uses the both extension methods. For example, `PropagationPolicy` and `ResourceBinding` use the `Custom Resource`, and `Cluster` resource uses the `Kubernetes API Aggregation Layer`.
+
+Therefore, `Cluster` resources do not have a CRD YAML file, and they are not visible when you execute the `kubectl get crd` command.
+
+So, why do we use the `Kubernetes API aggregation layer` to extend `Cluster` resources instead of using `custom resources`?
+
+This is because we need to set up the `proxy` sub-resource for the `Cluster` resource. By using `proxy`, users can access resources in member clusters. For details, please refer to [Aggregation Layer APIServer](https://karmada.io/ en/docs/next/userguide/globalview/aggregated-api-endpoint). Currently, the `custom resource` does not support setting `proxy` sub-resources, which is why we did not choose it.
```

Comment:
```suggestion
This is because the `Cluster` resource requires the setup of the `proxy` sub-resource. By using `proxy`, you can access resources in member clusters. For more information, refer to [Aggregation Layer APIServer](https://karmada.io/docs/next/userguide/globalview/aggregated-api-endpoint/). At present, Custom Resources do not support configuring `proxy` sub-resources, which is why it was not chosen for this purpose.
```